src/components/footer/Footer.tsx

```diff
@@ -0,0 +1,73 @@
+import cn from 'classnames';
+import { Todo } from '../../types/Todo';
+import { Select } from '../../types/Select';
+
+type Props = {
+  filterBy: string,
+  setFilterBy: (string: Select) => void,
+  todoList: Todo[];
+};
+
+const { All, Active, Completed } = Select;
+
+export const Footer: React.FC<Props> = ({
+  filterBy,
+  setFilterBy,
+  todoList,
+}) => {
+  function setLeftItems() {
+    let count = 0;
+
+    todoList.forEach(todo => {
+      if (!todo.completed) {
+        count += 1;
+      }
+    });
+
+    return count;
+  }
```

Comment:
try to calculate the number by using the `filter` method and `length`. use `useMemo` hook